python/ray/tune/tests/test_multinode_sync.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import time
+import unittest
+
+import ray
+from ray.autoscaler._private.fake_multi_node.test_utils import DockerCluster
+
+
+@ray.remote
+def remote_task(val):
+    return val
+
+
+class MultiNodeSyncTest(unittest.TestCase):
+    def setUp(self):
+        self.cluster = DockerCluster()
+        self.cluster.setup()
+
+    def tearDown(self):
+        self.cluster.stop()
+        self.cluster.teardown()
+
+    def testClusterAutoscaling(self):
+        """Sanity check that multinode tests with autoscaling are working"""
+        self.cluster.update_config({
+            "provider": {
+                "head_resources": {
+                    "CPU": 4,
+                    "GPU": 0
+                }
+            },
+        })
+        self.cluster.start()
+        self.cluster.connect(client=True, timeout=120)
```

Comment:
so if this is "client=False", what happens?

Reply:
Then it would connect to the Ray cluster via GCS (i.e. not using Ray client)

Reply:
Added a docstring